src/main.cpp

```diff
@@ -11,63 +12,72 @@ ESPCAN can_bus{};
 #endif
 
 #define WHEELS
-#define BRAKE_PRESSURE
+// #define BRAKE_PRESSURE
+
+constexpr uint8_t potentiometer_pin{34};
+VirtualTimerGroup timer_group;
 
 #ifdef WHEELS
-CANSignal<float, 0, 16, CANTemplateConvertFloat(0.1), CANTemplateConvertFloat(0)> fl_wheel_speed;
-CANSignal<float, 0, 16, CANTemplateConvertFloat(0.1), CANTemplateConvertFloat(0)> fr_wheel_speed;
-CANSignal<float, 0, 16, CANTemplateConvertFloat(0.1), CANTemplateConvertFloat(0)> bl_wheel_speed;
-CANSignal<float, 0, 16, CANTemplateConvertFloat(0.1), CANTemplateConvertFloat(0)> br_wheel_speed;
+CANSignal<float, 0, 16, CANTemplateConvertFloat(0.1), CANTemplateConvertFloat(0), false, ICANSignal::ByteOrder::kBigEndian> fl_wheel_speed;
```

Comment:
Why are we using big endian? I thought the default for the car was going to be little endian